internal/output/resource_output_test.go

```diff
@@ -132,9 +132,11 @@ func TestCmdOutput(t *testing.T) {
 			})
 
 			t.Run("with a key", func(t *testing.T) {
-				t.Skip()
 				t.Run("returns a success message", func(t *testing.T) {
 					expected := "Successfully deleted test-key"
+					input := `{
+						"key": "test-key"
+					}`
```

Comment:
err i'm confused, won't we not even have the key in the response body? or should I be shoving that in a response body when it's a delete request.

Reply:
It will be one of the flag's passed in to the command, so we can ignore the response body and pass in the given key instead.